views/pages/editQuantity.ejs

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
```

Comment:
Great work! 

It's good that you created a separate page for edit quantity for this sprint in order to keep track of all functionalities. In the future sprints what we want to do is maybe make this a pop up that allows the user to edit quantity on the same web page instead of navigating to another page. This can be done as we had it in the design where we have an edit icon beside each product in the fridge. Is that what you plan on doing?

In terms of number of lines, maybe next time to reduce these number of lines, just keep one line of space between different actions. That way it is easier for anyone to check the code and have it all in one view.

Hope this makes sense! Otherwise feel free to let me know and I'll be happy to explain further.